src/transactions.ts

```diff
@@ -133,9 +132,19 @@ export class Transaction {
       TxnState.STARTING_TRANSACTION,
       TxnState.TRANSACTION_IN_PROGRESS
     ];
+
     return activeStates.includes(this.state);
   }
 
+  get isCommitted(): boolean {
+    const committedStates: TxnState[] = [
```

Comment:
Can we move this definition to the top level of the file?

Reply:
Are you actually asking that all the getters defined are in alphabetical order? It's sitting after the `isActive` getter so I assumed that would be a natural place for it.

Reply:
Sorry for being unclear, I meant moving the `committedStates` constant out of the getter, it seems like something that could go at the top of the file as an UPPER_CASE const. Total nit, feel free to disregard as well if you think this makes it more readable.

Reply:
Ah. Nice suggestion. I've refactored that now and also with `isActive` as well.